tests/detectors/test_detectors_injection.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-FileCopyrightText: Portions Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import garak.attempt
+import garak.payloads
+import garak.detectors.base
+import garak.probes.injection
+from garak import _plugins
+
+
+def test_load_jinja_detector():
+    d = _plugins.load_plugin("detectors.injection.JinjaTemplateDetector")
+    assert isinstance(d, garak.detectors.base.Detector)
+
+
+def test_load_python_detector():
+    d = _plugins.load_plugin("detectors.injection.PythonCodeExecution")
+    assert isinstance(d, garak.detectors.base.Detector)
+
+
+def test_load_sqli_success_detector():
+    d = _plugins.load_plugin("detectors.injection.SQLiSuccess")
+    assert isinstance(d, garak.detectors.base.Detector)
+
+
+def test_load_sqli_echo_detector():
+    d = _plugins.load_plugin("detectors.injection.SQLiEcho")
+    assert isinstance(d, garak.detectors.base.Detector)
+
+
+def test_jinja():
+    d = _plugins.load_plugin("detectors.injection.JinjaTemplateDetector")
+    payload_template = (
+        garak.probes.injection.JinjaTemplatePythonInjection.payload_template
+    )
+    payload = payload_template.format(
+        "__import__('os').system('touch /tmp/garak.pwnd')"
+    )
+    a = garak.attempt.Attempt()
+    a.prompt = "test prompt"
+    a.outputs = [payload]
+    assert d.detect(a) == [1.0], f"{payload} contains a valid Jinja template"
```

Comment:
Does this message tell the reader what went wrong? maybe 
```suggestion
    assert d.detect(a) == [1.0], f"'{payload}' contains a matching Jinja template, which should give a hit"
```